tests/framework/extensions/clusters/clusters.go

```diff
@@ -78,3 +102,34 @@ func NewRKE2ClusterConfig(clusterName, namespace, cni, cloudCredentialSecretName
 
 	return v1Cluster
 }
+
+func CreateRKE2Cluster(client *rancher.Client, rke2Cluster *apisV1.Cluster) (*apisV1.Cluster, error) {
+	clusterCleanup := func() error {
+		err := client.Provisioning.Clusters(rke2Cluster.Namespace).Delete(context.TODO(), rke2Cluster.GetName(), metav1.DeleteOptions{})
+		if err != nil {
+			return err
+		}
+
+		watchInterface, err := client.Provisioning.Clusters(rke2Cluster.Namespace).Watch(context.TODO(), metav1.ListOptions{
+			FieldSelector:  "metadata.name=" + rke2Cluster.GetName(),
+			TimeoutSeconds: &defaults.WatchTimeoutSeconds,
+		})
+
+		if err != nil {
+			return err
+		}
+
+		return wait.WatchWait(watchInterface, func(event watch.Event) (ready bool, err error) {
+			if event.Type == watch.Error {
+				return false, fmt.Errorf("there was an error deleting cluster")
+			} else if event.Type == watch.Deleted {
+				return true, nil
+			}
+			return false, nil
+		})
+	}
```

Comment:
```suggestion
	})
```

Reply:
This brace doesn't need the parentheses, it closes the brace on line 107

Reply:
fixed